src/ImageSharp/Formats/Gif/GifEncoderCore.cs

```diff
@@ -154,18 +154,14 @@ private int GetTransparentIndex<TPixel>(QuantizedImage<TPixel> quantized)
         {
             // Transparent pixels are much more likely to be found at the end of a palette
             int index = -1;
+            var trans = default(Rgba32);
             for (int i = quantized.Palette.Length - 1; i >= 0; i--)
             {
-                quantized.Palette[i].ToXyzwBytes(this.buffer, 0);
+                quantized.Palette[i].ToRgba32(ref trans);
```

Comment:
Cool to see more of these dropped!

Reply:
Yeah, I wanna replace all of them and remove the method very soon.